spec/models/review_spec.rb

```diff
@@ -8,11 +8,14 @@
   it { should respond_to :rating }
   it { should validate_presence_of :content }
   it { should validate_presence_of :rating }
+  it { should validate_presence_of :video }
 
-  context 'for rating values between 0 and 5' do
+  let(:video) { video = Fabricate(:video) }
+
+  context 'for rating values between 1 and 5' do
```

Comment:
You could use at your advantage shoulda matchers - and validate inclusion of. Check out should matchers docs for this and also Rails validations guides to see validation on inclusion. Basically you select an array of options that can be only valid for the column :) hope this helps you.